pom.xml

```diff
@@ -37,22 +37,21 @@
     <properties>
         <!-- Versioning -->
         <version.arquillian.core>1.1.11.Final</version.arquillian.core>
-        <version.com.opera>1.5</version.com.opera>
         <version.phantom.driver>1.2.1.Final</version.phantom.driver>
-        <version.selenium>2.53.1</version.selenium>
-        <version.htmlunit.driver>2.20</version.htmlunit.driver>
+        <version.selenium>3.0.1</version.selenium>
+        <version.htmlunit.driver>2.23</version.htmlunit.driver>
         <version.junit>4.11</version.junit>
```

Comment:
well, yes we can but... in the gradle configuration we are testing against all combination of arquillian-core since 1.1.4.Final; and in the versions 1.1.4 and 1.1.5, there is an issue (when you use junit 4.12) related to an annotation @InSequence. 
So we can, but we need to limit the combination of arquillian-core versions we are testing with. There are two questions now:
1) do we need to test it with these two old versions of arq-core?
2) what would be the improvement when we used junit 4.12 ?